### DISCUSSION TITLE
Slim template

### DISCUSSION THREAD

**Discussion by DeCaelo** [ElMassimo/vite-plugin-full-reload#12] [Q&A] [answered]
hello, does this library work with slim template ?

**Comment by ElMassimo** [comment, marked as answer]
Hi Ludovic!

This library sets up file listeners, so it would work with any templates.

If you wanted to restrict to certain file extensions, you can do so: `'app/views/**/*.slim'`.

**Comment by DeCaelo** [comment]
weird, my page doesn't refresh

**Reply by ElMassimo** [reply]
If you share a minimal reproduction I can take a look.

Also, verify that the file paths you provide to the config are relative to the project root.
